tests/eosio.system_tests.cpp

```diff
@@ -1187,7 +1187,7 @@ BOOST_FIXTURE_TEST_CASE( proxy_actions_affect_producers, eosio_system_tester, *
 
 BOOST_FIXTURE_TEST_CASE(producer_pay, eosio_system_tester, * boost::unit_test::tolerance(1e-10)) try {
 
-   const double continuous_rate = 4.879 / 100.;
+   const double continuous_rate = 0.00995;
    const double usecs_per_year  = 52 * 7 * 24 * 3600 * 1000000ll;
    const double secs_per_year   = 52 * 7 * 24 * 3600;
```

Comment:
not 100% related, but why this is calculating one day less?

365 x 24 x 3600 = 31536000
52 * 7 * 24 * 3600 = 31449600